types/index.d.ts

```diff
@@ -608,7 +608,7 @@ declare namespace postgres {
     unsafe<T extends any[] = (Row & Iterable<Row>)[]>(query: string, parameters?: SerializableParameter[], queryOptions?: UnsafeQueryOptions): PendingQuery<AsRowList<T>>;
     end(options?: { timeout?: number }): Promise<void>;
 
-    listen(channel: string, cb: (value: string) => void): ListenRequest;
+    listen(channel: string, cb: (value: string) => void, onlisten?: () => void): ListenRequest;
```

Comment:
Would it make sense to rename the first callback, for clarity?

```suggestion
    listen(channel: string, onnotify: (value: string) => void, onlisten?: () => void): ListenRequest;
```

Reply:
Sounds good, applied suggestion 👍